crates/ward_ctl/src/main.rs

```diff
@@ -338,24 +285,13 @@ WantedBy=multi-user.target
 
 #[cfg(target_os = "linux")]
 fn uninstall_systemd_service() -> Result<()> {
-  use std::fs;
-
-  let user_service_dir = dirs::config_dir()
-    .unwrap_or_else(|| PathBuf::from("."))
-    .join("systemd/user");
-
-  let service_path = user_service_dir.join("ward.service");
-
-  if service_path.exists() {
-    fs::remove_file(&service_path)?;
-    println!("Systemd service removed.");
-    println!();
-    println!("To complete uninstallation:");
-    println!("  systemctl --user stop ward");
-    println!("  systemctl --user daemon-reload");
-  } else {
-    println!("No systemd service found at {:?}", service_path);
-  }
+  println!("Linux service uninstallation:");
+  println!();
+  println!("Run:");
+  println!("  systemctl --user stop ward");
+  println!("  systemctl --user disable ward");
+  println!("  rm ~/.config/systemd/user/ward.service");
+  println!("  systemctl --user daemon-reload");
 
   Ok(())
```

Comment:
remove this or make optional features, we need to use this with openrc, dinit, runit etc (systemd-less)